packages/shared/lib/models/NangoConfig.ts

```diff
@@ -139,4 +139,5 @@ export interface StandardNangoConfig {
     provider?: string;
     syncs: NangoSyncConfig[];
     actions: NangoSyncConfig[];
+    [`on-events`]: NangoSyncConfig[];
```

Comment:
Note that there is more work to make `on-events` script first-class citizen like syncs or actions. They are not shown in the UI, they are not supported for pre-built, etc... To keep in mind when we are gonna need to add support for those things cc @bastienbeurier 
For now I just needed to to add an attribute to `StandardNangoConfig`. I would always be empty though.